packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -270,7 +281,7 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
   // Allow the user to scroll within the modal but not on the body
   private _allowScrollOnModal = (elt: HTMLDivElement | null): void => {
     if (elt) {
-      allowScrollOnElement(elt, this._events);
+      allowScrollOnElement(elt, this._events, this._allowIosBodyScroll);
```

Comment:
What's missing from the current allowScrollOnElement that misses the touch capability? Instead of making a separate, new method to handle touch, can we fill in the missing gaps to handle touch in allowScrollOnElement?

Reply:
allowScrollOnElement allows scrolling of content within the element and when top or botton of content is reached, events are blocked, this way it prevents overscrolling [_preventOverscrolling](https://github.com/OfficeDev/office-ui-fabric-react/blob/2894062b1d0515055c29061dfc6f586f918ca41c/packages/utilities/src/scroll.ts#L34) where body would be scrolled. But this creates an issue, because _preventOverscrolling blocks all touch events . Example is my original issue #10982 where if you are zoomed in on mobile device, you cant pan around because of _preventOverscrolling event handler. We need to somehow allow the user to scroll the content, but also be able to pan around while zoomed in.
My easiest and minimal to implement solution was, to just make blocking happening in _preventOverscrolling optional, that what boolean argument allowIosOverscroll is used for.

Reply:
With this PR, i could create Panel and Modal components that dont block touch events. This is not perfect solution, as it allows body scrolling behind the overlay, which is a bit irritating, but UI is still intuitively usable. Where in current state, user is stuck in not usable UI.